.github/workflows/onReleasePublished.yml

```diff
@@ -0,0 +1,37 @@
+name: Update version in the docs after a release is published
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  trigger-doc-workflow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Check if release is a pre-release
+        id: check_prerelease
+        run: |
+          if [ "${{ github.event.release.prerelease }}" = "true" ]; then
+            echo "This is a pre-release, exiting."
+            exit 1
+          fi
+
+      - name: Get the published version
+        id: get_version
+        run: |
+          NEW_VERSION=${{ github.event.release.tag_name }}
+          NEW_VERSION=$(echo "${NEW_VERSION}" | sed 's/^v//')
+          echo "NEW_VERSION=${NEW_VERSION}" >> $GITHUB_OUTPUT
+
+
+      - name: Trigger WasabiDoc workflow
+        run: |
+          curl -X POST \
+          -H "Authorization: token ${{ secrets.WASABI_DOC_PAT }}" \
```

Comment:
ẀASABI_DOC_PAT` is not a good name because it was impossible to me to understand until I read the PR. I would use the word `TOKEN` .